crates/sui-json-rpc/src/api/write.rs

```diff
@@ -55,8 +62,11 @@ pub trait WriteApi {
         epoch: Option<BigInt<u64>>,
     ) -> RpcResult<DevInspectResults>;
 
-    /// Return transaction execution effects including the gas cost summary,
-    /// while the effects are not committed to the chain.
+    /// Simulate running the transaction, including all normally applied
+    /// checks, without actually running it. This is useful for estimating
+    /// the gas fees of a transaction before running it, but can also be used
+    /// to determine the side-effects of a transaction without actually running
+    /// it.
```

Comment:
```suggestion
    /// Simulate running the transaction, including all standard checks, 
    /// without actually running it. This is useful for estimating
    /// the gas fees of a transaction before executing it. You can also use it to 
    /// identify any side-effects of a transaction before you execute it on the network.
```

Reply:
Can we avoid passive voice?
`dryRun` can be used to estimate -> use `dryRun` to estimate 

Please don't use dry-running as it can be difficult to localize. The effects of running a transaction after you use `dryRun` might differ if...were modified after you used `dryRun`.